### ISSUE TITLE
Autogenerated code with the same function name will overwrite other class definition files

### ISSUE THREAD

**Issue by tjayrush** [Great-Hill-Corporation/quickBlocks#47]
In monitors especially if two smart contracts have similar function names.

https://github.com/Great-Hill-Corporation/ethslurp/issues/124

**Comment by tjayrush** [comment]
Closing for now. Will reopen once 0.2.0 is finished. Referenced by #293.

**Comment by tjayrush** [comment]
There is a #define in grabABI.cpp that turns on an experiment that solves this issue. I think it works, but I've not checked that it works. Turn it on, and check that it works.

**Comment by tjayrush** [comment]
This is a duplicate of #54
